components/Common/ThemeToggle/index.module.css

```diff
@@ -0,0 +1,8 @@
+.themeToggle {
+  svg {
+    @apply h-5
```

Comment:
Don't forget the indentation... Please...

Reply:
Right, but Extension has nothing to do with indentation. I guess how the colour can be confusing. But ultimately you can see these on git diffs.

Reply:
> git diffs

I never take really attention to that . Thank you for opening an eye for me

Reply:
Of course :) here to help! 🙇